<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RawViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="fileParameters">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>File</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="lineFileName"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushNext">
         <property name="text">
          <string>&amp;Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushPrevious">
         <property name="text">
          <string>&amp;Previous</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="EditingParamters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Edit</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupCenter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>&amp;Center</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinCenterX">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Y=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinCenterY">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupCrop">
         <property name="toolTip">
          <string notr="true">Crop image</string>
         </property>
         <property name="title">
          <string>Cro&amp;p</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>R=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinRcrop">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupRotate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Rotate</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkRot90">
            <property name="text">
             <string>90 deg</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Degree</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinRotateDeg">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupInvert">
         <property name="title">
          <string>In&amp;vert</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ViewingParameters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkAxisLim">
         <property name="text">
          <string>Hold view</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupColormap">
         <property name="title">
          <string>Color map</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboColormap">
            <item>
             <property name="text">
              <string>jet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>seismic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>coolwarm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RdBu</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RdYlBu</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bone</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gray</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>copper</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pink</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hsv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cool</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>spring</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>summer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>autumn</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>winter</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkNormColormap">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Norm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLogColormap">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupColorscale">
         <property name="title">
          <string>Co&amp;lor scale</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>min=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinCSmin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>max=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinCSmax">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupGrids">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Grids</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkCenterXYLines">
            <property name="text">
             <string>Center lines</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Calib. const.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinCalibConst">
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
            <property name="value">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Egrid=</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEGridVals">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>pticks=</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="linepTickVals"/>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkXYgrid">
            <property name="text">
             <string>X/Y grid</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_ile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionOpen_Next"/>
    <addaction name="actionOpen_Previous"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuF_ile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string notr="true">&amp;Open</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Center</string>
   </property>
  </action>
  <action name="action_Rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Rotate</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Close_2">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Grid">
   <property name="text">
    <string>&amp;Grid</string>
   </property>
  </action>
  <action name="actionOpen_Next">
   <property name="text">
    <string notr="true">Open &amp;Next</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Previous">
   <property name="text">
    <string notr="true">Open &amp;Previous</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionCro_p">
   <property name="text">
    <string>Cro&amp;p</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineFileName</tabstop>
  <tabstop>pushNext</tabstop>
  <tabstop>pushPrevious</tabstop>
  <tabstop>groupCenter</tabstop>
  <tabstop>spinCenterX</tabstop>
  <tabstop>spinCenterY</tabstop>
  <tabstop>groupCrop</tabstop>
  <tabstop>spinRcrop</tabstop>
  <tabstop>groupRotate</tabstop>
  <tabstop>checkRot90</tabstop>
  <tabstop>spinRotateDeg</tabstop>
  <tabstop>checkAxisLim</tabstop>
  <tabstop>comboColormap</tabstop>
  <tabstop>checkNormColormap</tabstop>
  <tabstop>checkLogColormap</tabstop>
  <tabstop>groupColorscale</tabstop>
  <tabstop>spinCSmin</tabstop>
  <tabstop>spinCSmax</tabstop>
  <tabstop>groupGrids</tabstop>
  <tabstop>checkCenterXYLines</tabstop>
  <tabstop>checkXYgrid</tabstop>
  <tabstop>spinCalibConst</tabstop>
  <tabstop>lineEGridVals</tabstop>
  <tabstop>linepTickVals</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
